src/saturn_engine/worker_manager/context.py

```diff
@@ -23,6 +23,8 @@ def static_definitions(self) -> StaticDefinitions:
         return self._static_definitions
 
     def load_static_definition(self, session: AnySession) -> None:
+        if not self.config.static_definitions_jobs_selector:
```

Comment:
I feel like the check should be done in `_load_static_definition`, and we should check 

```
if not config.static_definitions_directories:
  return None
definitions = load_definitions_from_paths(config.static_definitions_directories)
...
```

Reply:
Done